### DISCUSSION TITLE
Android 4.4.2 Does not display the accept call buttons

### DISCUSSION THREAD

**Discussion by chernovvdima** [evilcorpltd/aTox#1163] [Q&A] [answered]
![Screenshot_2023-09-03-15-19-57-234_com miui home](https://github.com/evilcorpltd/aTox/assets/143943095/dc65a54a-0cef-4a37-8737-cb94967b8d98)
Android 4.4.2 Does not display the accept call buttons. (MIUI-8.2.1.0). Is this a problem in the program or in the firmware ?

**Comment by robinlinden** [comment, marked as answer]
Looks like a problem with MIUI:
![call accept and reject buttons](https://github.com/evilcorpltd/aTox/assets/8304462/bf1cb51c-2de8-4b84-85fd-0df92e3cda06)

They're there for me when I test on a vanilla Android 4.4.2 ROM.

**Reply by chernovvdima** [reply]
Understood. So I'll pick up another firmware. Thanks!

**Reply by robinlinden** [reply]
Happy to help! Thanks for reaching out. :)
